scripts/vim-patch.sh

```diff
@@ -36,17 +36,22 @@ check_executable() {
 }
 
 get_vim_sources() {
-  check_executable hg
+  check_executable git
 
   echo "Retrieving Vim sources."
   if [[ ! -d ${VIM_SOURCE_DIR} ]]; then
     echo "Cloning Vim sources into '${VIM_SOURCE_DIR}'."
-    hg clone https://code.google.com/p/vim "${VIM_SOURCE_DIR}"
+    git clone https://github.com/vim/vim.git "${VIM_SOURCE_DIR}"
```

Comment:
It saves ~20M space, and it almost halves the clone time.  Added in next commit.